kivy/core/video/video_ffpyplayer.py

```diff
@@ -255,11 +255,13 @@ def _next_frame_run(self):
         while not self._ffplayer_need_quit:
             seek_happened = False
             if seek_queue:
-                vals = seek_queue[:]
```

Comment:
The reason for copy and then delete is because of threading issues. If you get the last value and then delete all elements, because this runs in a second thread, in between the get and delete a new seek may have been added from the other thread. So you end up losing that seek request. The way it was before (copy and then delete the number of copied elements) is thread safe so please change it back to that.